bench/wtperf/wtperf.c

```diff
@@ -603,7 +604,20 @@ worker(void *arg)
 op_err:			lprintf(cfg, ret, 0,
 			    "%s failed for: %s, range: %"PRIu64,
 			    op_name(op), key_buf, wtperf_value_range(cfg));
-			goto err;
+			/*
+			 * If we get a rollback error we rollback and continue
```

Comment:
This comment deserves expansion. I'd like it to mention:
* We only expect to see WT_ROLLBACK if explicit transactions are configured.
* This potentially changes the characteristics of the data set WiredTiger generates. If we don't retry operations after a rollback the data range can have gaps (random-range mode is the other case where this is true).